docs/_includes/page-notice.html

```diff
@@ -103,6 +103,50 @@ <h2 class="page-notice__title">Opt into eBay payments before Jan 12th to pay no
 </section>
     {% endhighlight %}
 
+    <h3 id="page-notice-dismissable-title">Dismissable Page Notice With Title</h3>
+    <section class="page-notice page-notice--information" role="region" aria-label="Information">
+        <div class="page-notice__header">
+            <svg class="icon icon--information-filled" focusable="false" height="24" width="24" role="img" aria-label="Information">
+                {% include symbol.html name="information-filled" %}
+            </svg>
+        </div>
+        <div class="page-notice__main">
+            <h3 class="page-notice__title">Notice Title</h3>
+            <p>Opt into eBay payments before Jan 12th to pay no selling fees.</p>
+            <p class="page-notice__cta" tabindex="0"><a href="https://www.ebay.com">Opt in</a></p>
```

Comment:
The `tabindex` attribute shouldn't be there.

Also, do we need the p tag at all? Could possibly drop it altogether?

Reply:
Yep, `tabindex` slipped through the cleanup cracks, fixed it.

If we can be certain that there will only be one `cta` button/link, then I suppose we don't need it. I can remove it if we're sure there won't be others.